cleanlab/filter.py

```diff
@@ -398,6 +411,80 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
+def _find_label_issues_multilabel(
+    labels,
+    pred_probs,
+    return_indices_ranked_by=None,
+    rank_by_kwargs={},
+    filter_by="prune_by_noise_rate",
+    frac_noise=1.0,
+    num_to_remove_per_class=None,
+    min_examples_per_class=1,
+    confident_joint=None,
+    n_jobs=None,
+    verbose=False,
+):
+    """
+    NOT
```

Comment:
you'll want to add documentation here, especially because some of the args now have particular shapes they require